ui/src/shared/components/SingleStat.tsx

```diff
@@ -41,6 +42,7 @@ interface Props {
   data: TimeSeriesServerResponse[] | FluxTable[]
   dataType: DataType
   onUpdateCellColors?: (bgColor: string, textColor: string) => void
+  onUpdateVisType?: TimeMachineContainer['handleUpdateType']
```

Comment:
I found this a little confusing to read. What would you think of just writing out type explicitly? hrrm.. I guess that's how the type is defined in a number of places. I still don't love it.

Reply:
agreed 👍